x/crisis/keeper/keeper.go

```diff
@@ -74,8 +74,9 @@ func (k Keeper) AssertInvariants(ctx sdk.Context) {
 
 	start := time.Now()
 	invarRoutes := k.Routes()
-
-	for _, ir := range invarRoutes {
+	fmt.Println("invarRoutes #", len(invarRoutes))
```

Comment:
Ditto - errors and diagnostic messages should go to `/dev/stderr`.

Reply:
will remove it once I will get a green light that we are OK with the "normal" changes.